Dockerfile_checkM

```diff
@@ -0,0 +1,66 @@
+FROM kbase/sdkbase2:python
```

Comment:
This loads the prerequisites (checkM, python modules). I've pushed the resulting image to dockerhub -- it speeds up development a lot if you don't have to wait for the docker image to build locally / on GitHub / on the kbase servers.